src/pages/ConnectWallet/WalletButton.tsx

```diff
@@ -104,16 +120,17 @@ export const WalletButton = ({
               />
             )}
           </StyledText>
+          // TODO: Finish search for className=".*[mp][lr][\d]*"
```

Comment:
Oops

<img width="1303" alt="image" src="https://user-images.githubusercontent.com/9263930/220888463-fc5df45a-1b47-4721-87bd-f65c7bdc47c3.png">

```suggestion
          {/* TODO: Finish search for className=".*[mp][lr][\d]*" */}
```